gcc/rust/hir/tree/rust-hir-expr.h

```diff
@@ -533,8 +539,10 @@ class LazyBooleanExpr : public OperatorExpr
   void accept_vis (HIRFullVisitor &vis) override;
   void accept_vis (HIRExpressionVisitor &vis) override;
 
-  Expr *get_lhs () { return main_or_left_expr.get (); }
+  void visit_lhs (HIRFullVisitor &vis) { main_or_left_expr->accept_vis (vis); }
+  void visit_rhs (HIRFullVisitor &vis) { right_expr->accept_vis (vis); }
 
+  Expr *get_lhs () { return main_or_left_expr.get (); }
```

Comment:
I think this is another case where we should be returning std::unique_ptr

Reply:
ack